### ISSUE TITLE
Revisit Obsolete User Table Data [Research Spike]

### ISSUE THREAD

**Issue by brian-ruf** [GSA/fedramp-automation#902]
### This is a ...

research - something needs to be investigated

### This relates to ...

- [ ] the **FedRAMP OSCAL baselines**
- [X] the **FedRAMP SSP OSCAL Example**
- [ ] the **FedRAMP SAP OSCAL Example**
- [ ] the **FedRAMP SAR OSCAL Example**
- [ ] the **FedRAMP POA&M OSCAL Example**
- [ ] the **FedRAMP OSCAL Validations**
- [ ] the **Not sure**

### User Story

As an author of OSCAL-based FedRAMP SSP's, I need to understand how the dropping of the Rev 4 user table from the Rev 5 Word template impacts information requirements relative to users.

### Background
The OSCAL SSP's `//system-implementation/user` assembly was originally designed to work hand-in-hand with the `//metadata/role` assembly to capture additional role information that is only required in the context of the SSP. It was created with the FedRAMP use case in mind. 

The Rev 4 modeling included several props to capture information required by the User Table in the FedRAMP Rev 4 SSP Word template. 

The FedRAMP review guidance was that every role referenced as a "responsible role" in any control, must have an entry in the user table, where additional information told reviewers whether the role was internal/external, privileged/non-privileged, and provided additional detail relative to these points.

The FedRAMP Rev 5 SSP Word template drops the table; however, controls still have responsible roles, and Tables 6.1 (Leveraged Authorizations) and 7.1 (External Systems/Services and Interconnections) still require user roles to be identified.

While these roles have to be called out, there appears to be no additional information requirements surrounding these roles.

### Goals

- Determine what - if any - information responsibility Rev 5 SSP authors have relative to user roles and the OSCAL SSP `//system-implementation/user` assembly.

### Acceptance Criteria

- All FedRAMP Documents Related to OSCAL Adoption (https://github.com/GSA/fedramp-automation) affected by the changes in this issue have been updated.
- A Pull Request (PR) is submitted that fully addresses the goals of this User Story. This issue is referenced in the PR.

**Comment by aj-stein-gsa** [comment]
So the cardinality of `/ssp/system-implementation/user` is [1 to unbounded in the core `v1.1.2` models](https://github.com/usnistgov/OSCAL/blob/v1.1.2/src/metaschema/oscal_ssp_metaschema.xml#L580-L587), so is this something we can effectively work around without kludgey behavior?

**Comment by brian-ruf** [comment]
> So the cardinality of /ssp/system-implementation/user is [1 to unbounded in the core v1.1.2 models](https://github.com/usnistgov/OSCAL/blob/v1.1.2/src/metaschema/oscal_ssp_metaschema.xml#L580-L587), so is this something we can effectively work around without kludgey behavior?

Yes. We have a few options. We can still use the user table to capture internal and external users, or we can just have a single entry user table that has no data. The following is the valid minimum to satisfy the requirement:
```
      <user uuid="11111111-2222-4000-8000-008000000001">
         <title>Users</title>
      </user>
```

Unless a compelling reason emerges or leadership says otherwise, I am simply including only the above content in the example SSP for the `user` assembly, along with a remark that indicates it is under investigation for continued applicability.

**Comment by brian-ruf** [comment]
This issue is intertwined with issue #534
